src/themes/default/elements/flag.variables

```diff
@@ -17,19 +17,19 @@
 @flags: {
   @1f3f3-fe0f-200d-26a7-fe0f: {
     countrycode: transgender;
-    class: transgender;
+    class: false;
     aliasClass: false;
     aliasClass2: false;
   };
   @1f3f3-fe0f-200d-1f308: {
     countrycode: rainbow;
-    class: rainbow;
-    aliasClass: gay_pride;
+    class: gay_pride;
```

Comment:
any idea why this generated `i.flag.gay.pride::before` selector? (not tested this exact PR)

Reply:
Because the code assumes an underscrore to be a word/classname split. (Same for "cayman_islands") following the original idea of SUI to have "spoken sentences words as classnames"
However, i just named it "pride" and added another alias LGBT instead by bb3281735f72040b030e068a4d61ab208a51ee6c according to https://emojiterra.com/rainbow-flag/ or  https://emojipedia.org/rainbow-flag/